src/scene/lobby-scene.ts

```diff
@@ -23,20 +25,24 @@ export default class LobbyScene extends BaseScene<LobbySceneController> {
 
     create() {
         this.add.existing(new TilingBackground(this, 'lobby_bg', new Phaser.Math.Vector2(2, 1), 35, 1.75));
-        this.add.text(0, 0, `Lobby ( ${this.aznopoly.room.id} )`, FONT_STYLE_HEADLINE);
-        this.playerList = this.add.existing(new PlayerList(this, this.aznopoly.isHost, 100, 200, 450));
+        //this.add.image(0, 0, 'backdrop').setOrigin(0, 0).setAlpha(0.5, 0.5, 0.5, 0.5)
+        //this.add.existing(new AzNopolyBar(this,`Lobby ( ${this.aznopoly.room.id} )`, 50))
+        this.add.rectangle(0, 0, WIDTH, 100, COLOR_PRIMARY_2).setOrigin(0,0)
```

Comment:
Please use a AzNopolyBar instead, for a consistent Look&Feel throughout the whole application.
If you want to change the appearance, change it in there, so that every screen get's the same changes